protocol/source/served/lsp/protocol.d

```diff
@@ -3085,6 +3086,43 @@ struct ReferenceParams
 	ReferenceContext context;
 }
 
+@serdeFallbackStruct
+@allowedMethods("textDocument/inlayHint")
+struct InlayHintParams
+{
+	TextDocumentIdentifier textDocument;
+	TextRange range;
+}
+
+@serdeFallbackStruct
+struct InlayHint
+{
+    Position position;
+    Variant!(string, InlayHintLabelPart[]) label;
+    @serdeOptional Optional!InlayHintKind kind;
+    @serdeOptional Optional!(TextEdit[]) textEdits;
+    @serdeOptional Variant!(void, string, MarkupContent) tooltip;
+    @serdeOptional Optional!(bool) paddingLeft;
+    @serdeOptional Optional!(bool) paddingRight;
+    // TODO: @serdeOptional Optional!(LSPAny) data;
```

Comment:
LSPAny is referenced in the `metaModel.json` but couldn't find a corresponding reference in `serve-d`. I've verified that this PR works without it.

Reply:
you can use OptionalJsonValue here instead

Reply:
```suggestion
    OptionalJsonValue data;
```

Reply:
Fixed.